torch/ao/sparsity/sparsifier/base_sparsifier.py

```diff
@@ -42,41 +48,41 @@ class BaseSparsifier(abc.ABC):
         >>> # model.layer1.weight will have `sparsity_level` = 0.7 (getting default)
         >>> sparsifier = BaseSparsifier(config, defaults)
     """
-    def __init__(self, defaults):
+    def __init__(self, defaults: Optional[Dict[str, Any]] = None):
         super().__init__()
-        self.defaults = defaults
-        if self.defaults is None:
-            self.defaults = dict()
+        if defaults is None:
+            defaults = dict()
+        self.defaults: Dict[str, Any] = defaults
```

Comment:
nit: 
```
self.defaults: Dict[str, Any] = defaults or dict()
```

Reply:
that doesn't seem to cause an issue actually